### ISSUE TITLE
Data validation

### ISSUE THREAD

**Issue by dbGIS** [blueaccounting/charts#14]
When tributary data are missing an entry for a given year, the value list used to build the chart will be shorter than expected, causing values to shift. An added data validation step would allow a null value to be inserted for missing years.

**Comment by sfGLC** [comment]
If I understand the issue correctly, this issue is not about missing a value. The program expects a data entry for every year, even though the value for any given year could be empty. As long as there is data entry for the span of the period, e.g., 2008 - 2018, missing values will be filled as null and the values should not shift.

**Comment by dbGIS** [comment]
That is true. In our original mockup data there were missing years, causing the values to shift. I created this issue to prompt further thought on how we could prevent such issues. My thought was to explore a data validation step that would add a null value for missing years in the sequence or explore the idea of using CSVs as a data source; which would allow program staff to take more ownership of the data updates.
